src/test/java/com/fasterxml/jackson/databind/ObjectReaderTest.java

```diff
@@ -805,4 +808,111 @@ public int hashCode() {
             return _delegate.hashCode();
         }
     }
+
+
+    public void testReadValueFromFile() throws IOException {
```

Comment:
Could you change these to regular `Exception`s so merging to 3.0 (master) is easier (exceptions changed NOT to extend `IOException`).